swift/ios-shoppe-demo/Views/CartQuantityTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  File.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 4/9/20.
```

Comment:
headers !== FullStory

Reply:
Ok its hurt me enough lol I will make a PR for this to automatically be done 😭